final/index.html

```diff
@@ -0,0 +1,180 @@
+<!doctype html>
+<!--
+Copyright 2015 Google Inc. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Polymer Offline Weather</title>
+
+    <link rel="manifest" href="../common/manifest.json">
+    <meta name="theme-color" content="#006064">
+    <link rel="icon" href="../common/icon.png">
+
+    <script src="../bower_components/webcomponentsjs/webcomponents-lite.min.js"></script>
+    <link rel="import" href="../bower_components/geo-location/geo-location.html">
```

Comment:
you could make an elements.html for these imports:

    <link rel="import" href="../elements.html">

Then have that file reference the `bower_components` folder in the normal way.

This may be unnecessary though.